# contains sequence ID, time stamp, and frame ID.
Header header
# 1 dimensional array of pose_scores
float64[] pose_scores
# unpacked multi dimensional array of keypoint scores
float64[] keypoint_scores
# unpacked multi dimensional array of keypoint coordinates
float64[] keypoint_coords
